src/testdrive/src/util/postgres.rs

```diff
@@ -50,8 +51,11 @@ pub fn config_url(config: &Config) -> Result<Url, anyhow::Error> {
 
 pub async fn postgres_client(
     url: &str,
+    default_timeout: Duration,
```

Comment:
I think this needs to be named `connect_timeout` here so that it is explict that this timeout applies only to the connection attempt and nothing further.

In order to get a timeout that applies to the actual execution of the operation, tokio shenanigans need to be used (which we do at some places but maybe not everwhere).

Reply:
Now also used for query timeouts.